plugins/GrapesJsBuilderBundle/Tests/Unit/EventSubscriber/EmailSubscriberTest.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MauticPlugin\GrapesJsBuilderBundle\Tests\Unit\EventSubscriber;
+
+use Mautic\EmailBundle\Entity\Email;
+use Mautic\EmailBundle\Entity\EmailRepository;
+use Mautic\EmailBundle\Event\EmailEditSubmitEvent;
+use Mautic\EmailBundle\Helper\EmailConfigInterface;
+use Mautic\EmailBundle\Model\EmailModel;
+use MauticPlugin\GrapesJsBuilderBundle\Entity\GrapesJsBuilder;
+use MauticPlugin\GrapesJsBuilderBundle\Entity\GrapesJsBuilderRepository;
+use MauticPlugin\GrapesJsBuilderBundle\EventSubscriber\EmailSubscriber;
+use MauticPlugin\GrapesJsBuilderBundle\Integration\Config;
+use MauticPlugin\GrapesJsBuilderBundle\Model\GrapesJsBuilderModel;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class EmailSubscriberTest extends TestCase
+{
+    private Config|MockObject $config;
+    private GrapesJsBuilderModel|MockObject $grapesJsBuilderModel;
+    private GrapesJsBuilderRepository|MockObject $grapesJsBuilderRepo;
```

Comment:
It's weird. One thing I can think of is that the type should be `EmailModel&MockObject` instead but PHP 8.0 doesn't support that yet. Can you try if
```suggestion
    /** @var MockObject&GrapesJsBuilderRepository */
    private MockObject $grapesJsBuilderRepo;
```
would satisfy PHPSTAN?